### PR TITLE
Chore/sample vanilla app

### PR DESCRIPTION
**Pull request by Theeraphat-Sorasetsakul**

### CODE REVIEW

File: samples/vanilla/index.html [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        </div>
+        <div class="form-footer" slot="footer">
+          <ef-button id="confirm-button" cta>Confirm</ef-button>
+          <ef-button id="cancel-button">Cancel</ef-button>
+        </div>
+      </ef-dialog>
+    </div>
+    
+    <script>
+      const defaultTheme = document.documentElement.getAttribute('theme');
```
Comment (Jidapa-Pai) [inline samples/vanilla/index.html:107]: Can we move these JS code to `index.js`? I think it easier to read and maintenance.

File: samples/vanilla/index.html [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+            tab.active = false;
+          });
+        };
+
+        tabs.forEach((tab) => {
+          const type = tab.label.substring(0, tab.label.indexOf(' ')).toLowerCase();
+          if (chartType === type) {
+            tab.active = true;
+          }
+          tab.addEventListener('tap', function (e) {
```
Comment (Jidapa-Pai) [inline samples/vanilla/index.html:214]: The code has mixed function and arrow function. Can you make all function has the same code style?

**Review by Jidapa-Pai — COMMENTED** [review]
It would be good if you can group a functions together like event handler.

File: samples/vanilla/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Vanilla + Refinitiv-ui
+
+This project was generated without the use of framework.
+
+## Getting started
+
+Install dependencies and deduplicate packages.
+
+```sh
+  npm install && npm dedupe
```
Comment (wsuwt) [inline samples/vanilla/README.md:10]: I don't think we need to dedupe packages in V5.

File: samples/vanilla/package.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "main": "index.html",
+  "scripts": {
+    "start": "npm run build && webpack serve",
+    "build": "npm run build:light && npm run build:dark && webpack build --mode production",
+    "build:light": "lessc ./src/styles/light.less ./dist/styles/light.css",
+    "build:dark": "lessc ./src/styles/dark.less ./dist/styles/dark.css",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (wsuwt) [inline samples/vanilla/package.json:14]: Can you guys make sure that it is ok to use this license or is there more appropriate one ?

File: samples/vanilla/README.md (hunk 3 above)
Comment (Theeraphat-Sorasetsakul) [reply samples/vanilla/README.md:10]: Agree. I will remove it.

File: samples/vanilla/package.json (hunk 4 above)
Comment (Theeraphat-Sorasetsakul) [reply samples/vanilla/package.json:14]: Not sure about this. I'll delete it at first.

**Comment by Theeraphat-Sorasetsakul** [comment]
Move to PR instead.

**Pull request closed without merging**